docs/misc/config.md

```diff
@@ -51,6 +110,11 @@ The `ConfigValue` specific methods take in two additional components:
 - A validator to make sure the deserialized object is valid
 - A class representing the data type of the config value
 
+For lists, there are two additional components you can supply:
+
+- A supplier for new elements. This is used by the UI to allow the user adding new elements.
```

Comment:
This seems wrong. The supplier past in is the default list. Are you talking about the default value in a list of acceptable values?

Reply:
This one is new with https://github.com/neoforged/NeoForge/pull/1199

Sorry, should have linked that PR at the top, but it was kinda late and I was tired.